BEIMA.DB.Schemas/device-schema.txt

```diff
@@ -0,0 +1,25 @@
+{
+    "templateId": String,
```

Comment:
It seems like this field will refence a device template (device type) ID to determine the type of the device, but I think we also need a unique ID per device entry

Reply:
MongoDB will automatically populate a unique ObjectID whenever you insert into the database